modules/mono/SCsub

```diff
@@ -135,8 +135,8 @@ def mono_build_solution(source, target, env):
         msbuild_info = find_msbuild_windows()
         if msbuild_info is None:
             raise RuntimeError('Cannot find MSBuild executable')
-        msbuild_path = msbuild_windows[0]
-        framework_path_override = msbuild_windows[1]
+        msbuild_path = msbuild_info[0]
```

Comment:
I fixed that already in 23740c8a913c4a40618903707fe74a00fe44ad94.

Reply:
Oh, sorry. I should have checked whether there were new updates beforre submitting. Will rebase.